posthog/gzip_middleware.py

```diff
@@ -19,6 +19,21 @@ class ScopedGZipMiddleware(GZipMiddleware):
     see: https://docs.djangoproject.com/en/4.0/ref/middleware/#module-django.middleware.gzip
 
     Rather than solve for those across the whole app. We can add it to specific paths
+
+    http://breachattack.com/resources/BREACH%20-%20SSL,%20gone%20in%2030%20seconds.pdf
+
+    The vulnerability requires two things
+
+        • Reflect user-input in HTTP response bodies
+        • Reflect a secret (such as a CSRF token) in HTTP response bodies
+
+    e.g. a CSRF token in the URL and in the response body, or form input value in the request and in the response
+
+    If an API path does that, an attacker can use knowledge of the compression algorithm
+        to recover the secret from the compressed response
+
+    If a given API path doesn't do that it is safe to compress.
+        Add a pattern that matches it to GZIP_RESPONSE_ALLOW_LIST
```

Comment:
> If an API path does that, an attacker can use knowledge of the compression algorithm to recover the secret from the compressed response

Whoa. 🤯 

Could we reference this warning also in `settings/web.py`? I can imagine somebody adding a few more paths there, and not finding this text to know about the security problem.